run_kraken.py

```diff
@@ -203,9 +205,10 @@ def main(cfg):
                             )
                             sys.exit(1)
                         elif scenario_type == "arcaflow_scenarios":
```

Comment:
We will need to support telemetry for all the scenarios i.e telemetry as the positional arguments. Thoughts?